python/matplotlib.ipynb

```diff
@@ -1,748 +1,730 @@
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "Tabelle mit allen Farben und Styles: [matplotlib.axes.Axes.plot](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.plot.html)\n",
+                "\n",
+                "Neue Grenzen mit `xlim(a, b)` und `ylim(a, b)`"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "plt.plot(t, np.sin(t))\n",
+                "plt.xlim(0, 2 * np.pi)\n",
+                "plt.ylim(-1.2, 1.2)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Es fehlt noch etwas...\n",
+                "![XKCD comic on why you should label your axes.](http://imgs.xkcd.com/comics/convincing.png \"And if you labeled your axes, I could tell you exactly how MUCH better.\")"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "with plt.xkcd():\n",
+                "    plt.title('Axes with labels')\n",
+                "    plt.plot(t, np.sin(t))\n",
+                "    plt.xlabel('t / s')\n",
+                "    plt.ylabel('U / V')\n",
+                "    plt.ylim(-1.1, 1.1)\n",
+                "    plt.xlim(0, 2 * np.pi)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "Achsen-Beschriftungen können mit LaTeX-Code erstellt werden → LaTeX-Kurs in der nächsten Woche."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "plt.plot(t, np.sin(t))\n",
+                "plt.xlabel(r'$t / \\mathrm{s}$')\n",
+                "plt.ylabel(r'$U / \\mathrm{V}$')"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Einheiten in Achsenbeschriftungen werden wegdividiert:"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "plt.plot(t, np.sin(t))\n",
+                "plt.xlabel(r'$t / \\mathrm{s}$')\n",
+                "plt.ylabel(r'$U \\,/\\, \\mathrm{V}$')  # Spaces sind Geschmacksfrage……"
+            ]
```

Comment:
Collapse into one cell?

Reply:
Is the way alright, in which I did it now?